Makefile

```diff
@@ -2,14 +2,11 @@ all: build
 
 SHELL :=/bin/bash -euEo pipefail
 
-IMAGE_REF		?= docker.io/scylladb/scylla-operator:latest
+IMAGE_LATEST_TAG ?= latest
```

Comment:
nit: would it be better to call it just `IMAGE_TAG`? would still keep the meaning if someone overrides the value on invocation